src/test/java/com/wikia/webdriver/testcases/mercurytests/SearchMobileWikiTests.java

```diff
@@ -0,0 +1,226 @@
+package com.wikia.webdriver.testcases.mercurytests;
+
+import com.wikia.webdriver.common.contentpatterns.MercurySubpages;
+import com.wikia.webdriver.common.contentpatterns.MercuryWikis;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.skin.Skin;
+import com.wikia.webdriver.elements.mercury.pages.ArticlePage;
+import com.wikia.webdriver.elements.mercury.pages.SearchResultsPage;
+import org.testng.annotations.Test;
+
+@InBrowser(browser = Browser.CHROME)
+public class SearchMobileWikiTests extends SearchTests {
+
+  private static final String SEARCH_PHRASE = "Infobox";
+  private static final String SEARCH_PHRASE_NO_RESULTS = "AComplexQueryWithNoResults";
+  private static final String MULTIPLE_RESULTS_SEARCH_PHRASE = "Test";
+  private static final String SINGLE_RESULT_SEARCH_PHRASE = "SRPWithOnlyOneSearchResult";
+  private static final String EMPTY_SEARCH_PHRASE = "";
+  // Mobile Wiki tries to get 25 results, but a bug in the Search API returns 24 for the first batch
+  // See SUS-1151 for details
+  private static final int SEARCH_RESULTS_NUMBER_FIRST_BATCH = 24;
+  private static final int SEARCH_RESULTS_NUMBER_NEXT_BATCH = 25;
+
+  @Execute(onWikia = MercuryWikis.MERCURY_AUTOMATION_TESTING)
+  @InBrowser(emulator = Emulator.GOOGLE_NEXUS_5)
+  @Test(groups = {"mercury_search_navigateUsingSearchSuggestionsOnMobile",
+                  "Mercury_Search_001"})
+  public void mercury_search_navigateUsingSearchSuggestionsOnMobile() {
+    ArticlePage article = new ArticlePage().open(MercurySubpages.MAIN_PAGE);
+    String clickedSuggestion = article.getTopBar()
+        .openSearch()
+        .typeInSearch(SEARCH_PHRASE)
+        .clickSearchSuggestion(0, Skin.MOBILE_WIKI);
+
+    Assertion.assertEquals(
+        clickedSuggestion.toLowerCase(),
+        article.getHeader().getPageTitle().toLowerCase()
+    );
+  }
+
+  @Execute(onWikia = MercuryWikis.MERCURY_AUTOMATION_TESTING)
+  @InBrowser(emulator = Emulator.GOOGLE_NEXUS_5)
+  @Test(groups = {"mercury_search_navigateToPageUsingSearchResults", "Mercury_Search_001"})
```

Comment:
There is also `Mercury_Search_002`, @nikodamn split them in https://github.com/Wikia/selenium-tests/pull/1427, probably because 002 related to the SRP.